src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -56,7 +56,7 @@ public abstract partial class Constraint
         protected virtual System.Data.DataSet _DataSet { get { throw null; } }
         [System.ComponentModel.DefaultValueAttribute("")]
         public virtual string ConstraintName { get { throw null; } set { } }
-        [System.ComponentModel.Browsable(false)]
+        [System.ComponentModel.BrowsableAttribute(false)]
```

Comment:
This comes from how the tool generates them. We are trying to directly have the tool output so that can easily regenerate without much churn.